modules/user_manual/pages/files/access_webdav.adoc

```diff
@@ -100,10 +100,10 @@ The resulting dialog should appear with WebDAV already selected.
 . If WebDAV is not selected, select it.
 . Click btn:[Next].
 . Enter the following settings:
-* Name: The name you want to see in the *Places* bookmark, for example ownCloud.
-* User: The ownCloud username you used to log in, for example admin.
-* Server: The ownCloud domain name, for example *example.com* (without **http://** before or directories afterwards).
-* Folder: Enter the path `owncloud/remote.php/dav/files/USERNAME/`.
+* *Name:* The name you want to see in the *Places* bookmark, for example ownCloud.
+* *User:* The ownCloud username you used to log in, for example admin.
+* *Server:* The ownCloud domain name, for example *example.com* (without **http://** before or directories afterwards).
+* *Folder:* Enter the path `owncloud/remote.php/dav/files/USERNAME/`.
```

Comment:
regarding the url, pls check https://github.com/owncloud/docs/pull/1822

Reply:
Thanks for spotting that.